src/pretix/control/templates/pretixcontrol/pdf/index.html

```diff
@@ -472,6 +521,13 @@ <h2>{% trans "Welcome to the PDF ticket editor!" %}</h2>
                     </button>
                 </div>
             </form>
+            <p>&nbsp;</p>
+            <div class="alert alert-info" id="version-notice">
+                {% blocktrans trimmed with print_version="2.18" scan_version="1.22" %}
+                    This layout uses new features. If you print from your device, make sure you use pretixPRINT version
+                    {{ print_version }} (or newer) or pretixSCAN Desktop version {{ scan_version }} (or newer).
+                {% endblocktrans %}
+            </div>
```

Comment:
I would move that before `#preview-form` to force the user a little bit more to read it, but that is just personal preference.

Reply:
I tried that first, but didn't want to bother the "good users" (who update regularly) too much… I found no perfect place… :(